src/index.js

```diff
@@ -228,16 +259,18 @@ async function processVt(inputs, name, filePath) {
 /**
  * Get Release
  * @param {InstanceType<typeof github.GitHub>} octokit
+ * @param {String} release_id
  * @return {Promise<InstanceType<typeof github.GitHub>|Undefined>}
  */
-async function getRelease(octokit) {
-    if (!github.context.payload.release?.id) {
+async function getRelease(octokit, release_id) {
+    const target_release_id = release_id || github.context.payload.release?.id
```

Comment:
To allow a specific ID to be processed.
So it does not rely on a release event.